utility/BUILD.bazel

```diff
@@ -22,6 +25,10 @@ kt_android_library(
         "//app:crashlytics",
         "//app:crashlytics_deps",
         "//model",
+        "//utility/src/main/java/org/oppia/android/util/accessibility:accessibility_module",
+        "//utility/src/main/java/org/oppia/android/util/accessibility:android_accessibility_manager",
+        "//utility/src/main/java/org/oppia/android/util/accessibility:custom_accessibility_manager",
+        "//utility/src/main/java/org/oppia/android/util/accessibility:fake_accessibility_manager",
```

Comment:
Here & below: this one should only be included in test deps, not production classes.

Reply:
Done